### ISSUE TITLE
For `Bottom bar component`, update the animation of the `selected tab indicator`

### ISSUE THREAD

**Issue by pylapp** [Orange-OpenSource/ouds-flutter#633]
### Description

Today, the *tab bar component* displays a *selected tab indicator* in the top of the selected tab.
This indicator is animated: it moves simply from the old selected tab to the new one, with a system transition which looks like a simple slide from one tab to another.

The design team (cc @jerome-regnier) needs to update the animation as described bellow:
- no more slide from old selected tab to the new one
- when a tab moves from *selected* state to *unselected* state the indicator disappears
- when a tab moves from *unselected* state to *selected* state the indicator appears with the new animation: the separator is still in the same place in top of the tab but its line is drawn from the center of the tab and expands to the limits of the tab

For example, given:
- a tab T
- with X1 the X coordinate of the beginning of the tab (top)
- with X2 the X coordinate of the end of the tab (e.g. X1 + T width) (top)
- with Y the Y coordinate of the tab (top)
Then the animation starts at point `X1 + (T width) / 2)` and expands to points at (X1, Y) and (X2, Y)

See video attached
https://github.com/user-attachments/assets/be093524-eba6-427d-970b-a6edca65a0cf

### Motivation & Context

Bring a cooler animation, mainly for NDE project

**Comment by AhmedAmineZr** [comment]
📣 New Firebase App Distribution Alpha upload available 🚀 (Android)

Please, get the new build from App Tester app, or contact the project maintainers (@AhmedAmineZr ) to get access.

Display name: DesignToolbox (ALPHA) (701)
Version: 2.0.0
Build type: Alpha
Build details (GitHub): 633
Related Issues: #625 #627 
cc @B3nz01d (product owner)
cc @AhmedAmineZr @nouha06 (dev team)
cc @jerome-regnier (design team)
cc @pya35 (Accessibility)

**Comment by AhmedAmineZr** [comment]
📣 New TestFlight Alpha upload available 🚀 (iOS)

Please, get the new build from TestFlight app, or contact the project maintainers (@AhmedAmineZr) to get access.

Display name: Design Toolbox Flutter (701)
Version: 2.0.0
Build type: Alpha
Build details (GitHub): 633
Related Issues: https://github.com/Orange-OpenSource/ouds-flutter/issues/625 https://github.com/Orange-OpenSource/ouds-flutter/issues/627
cc @B3nz01d (product owner)
cc @AhmedAmineZr @nouha06 (dev team)
cc @jerome-regnier (design team)
cc @pya35 (Accessibility)

**Comment by AhmedAmineZr** [comment]
📣 New Firebase App Distribution Alpha upload available 🚀 (Android)

Please, get the new build from App Tester app, or contact the project maintainers (@AhmedAmineZr ) to get access.

Display name: DesignToolbox (ALPHA) (704)
Version: 2.0.0
Build type: Alpha
Build details (GitHub): 633
Related Issues: https://github.com/Orange-OpenSource/ouds-flutter/issues/625 https://github.com/Orange-OpenSource/ouds-flutter/issues/627
cc @B3nz01d (product owner)
cc @AhmedAmineZr @nouha06 (dev team)
cc @jerome-regnier (design team)
cc @pya35 (Accessibility)

**Comment by AhmedAmineZr** [comment]
📣 New TestFlight Alpha upload available 🚀 (iOS)

Please, get the new build from TestFlight app, or contact the project maintainers (@AhmedAmineZr) to get access.

Display name: Design Toolbox Flutter (704)
Version: 2.0.0
Build type: Alpha
Build details (GitHub): 633
Related Issues: https://github.com/Orange-OpenSource/ouds-flutter/issues/625 https://github.com/Orange-OpenSource/ouds-flutter/issues/627
cc @B3nz01d (product owner)
cc @AhmedAmineZr @nouha06 (dev team)
cc @jerome-regnier (design team)
cc @pya35 (Accessibility)

**Comment by B3nz01d** [comment]
@AhmedAmineZr On iOs the disapearing animation does not work. The line is switching from on to off without any animation

**Comment by jerome-regnier** [comment]
le marker me semble bon mais je n'ai pas la possibilité de voir le timing que tu as mis.
Est ce que tu as bien ces paramètres. ?

<img width="250" height="476" alt="Image" src="https://github.com/user-attachments/assets/21f68fd3-976c-4fef-bb33-9f37c72c1b4b" />

**Comment by AhmedAmineZr** [comment]
📣 New TestFlight Alpha upload available 🚀 (iOS)

Please, get the new build from TestFlight app, or contact the project maintainers (@AhmedAmineZr) to get access.

Display name: Design Toolbox Flutter (705)
Version: 2.0.0
Build type: Alpha
Build details (GitHub): 633
Related Issues:#625 #627
cc @B3nz01d (product owner)
cc @AhmedAmineZr @nouha06 (dev team)
cc @jerome-regnier (design team)
cc @pya35 (Accessibility)

**Comment by AhmedAmineZr** [comment]
📣 New Firebase App Distribution Alpha upload available 🚀 (Android)

Please, get the new build from App Tester app, or contact the project maintainers (@AhmedAmineZr ) to get access.

Display name: DesignToolbox (ALPHA) (705)
Version: 2.0.0
Build type: Alpha
Build details (GitHub): 633
Related Issues: #625 #627
cc @B3nz01d (product owner)
cc @AhmedAmineZr @nouha06 (dev team)
cc @jerome-regnier (design team)
cc @pya35 (Accessibility)

**Comment by jerome-regnier** [comment]
👌 OK Design
